control/poseidon.txt

```diff
@@ -15,7 +15,7 @@ queryserver_port=24390
 # to the poseidon operate properly !
 # The available server types for now are : Default, bRO_.* (check servertypes.txt)
 # You should modify this if you're having problems with char list.
-server_type=bRO_2016-11-08a
+server_type=Default
 
 # Char Block Size

```

Comment:
i think we should remove this. This parameter is not used anywhere

Reply:
fixed at https://github.com/OpenKore/openkore/pull/3468/commits/8b056b89b8e962d7c7c4bba7b1140705989af158